{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "a7f9897c_01207975",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1008672
      },
      "writtenOn": "2021-03-21T12:02:17Z",
      "side": 1,
      "message": "Why dhcp networks are not tested?",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "e1fc2c93_2c897a93",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1018675
      },
      "writtenOn": "2021-03-21T12:03:19Z",
      "side": 1,
      "message": "exactly what I wanted to ask",
      "parentUuid": "a7f9897c_01207975",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "d265b831_e365d63a",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1008484
      },
      "writtenOn": "2021-03-21T12:55:37Z",
      "side": 1,
      "message": "methos name is  check_guest_provider_networks\nsee this https://docs.openstack.org/networking-ovn/queens/admin/refarch/provider-networks.html",
      "parentUuid": "e1fc2c93_2c897a93",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "59ae7e6b_8b4dc69a",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1008672
      },
      "writtenOn": "2021-03-21T13:03:18Z",
      "side": 1,
      "message": "I read the doc, but I\u0027m not sure I\u0027m following.\n\nCurrently, sriov network created with \"dhcp: false\", but the instance still has those nics configured.\nIs there a reason not to check them?\nIt is our pure dataplane.",
      "parentUuid": "d265b831_e365d63a",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "25bf3578_ba1f1a41",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1008484
      },
      "writtenOn": "2021-03-21T14:19:51Z",
      "side": 1,
      "message": "It is pure data plane correct,\n1)The issue related to ping, is happening in mgmt network, only.\nIs management network is provider?\n\nSo if the purpose of the test is to check provider net, so by definition, there is no DHCP managed by neutron, it makes it tenant network.\n\nWe could use the same method and run it for provider and tenant network,\nI do not see it as an issue,\n\nMHO, Test coverage for tenant tested through ssh to vm",
      "parentUuid": "59ae7e6b_8b4dc69a",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "e05beb1f_6501c08b",
        "filename": "nfv_tempest_plugin/tests/scenario/base_test.py",
        "patchSetId": 4
      },
      "lineNbr": 97,
      "author": {
        "id": 1008672
      },
      "writtenOn": "2021-03-21T14:23:43Z",
      "side": 1,
      "message": "But the mgmt network created with dhcp enabled.\nDoes it makes it a provider network?\n\nWhy there is an issue at all?\nMaybe we need to understand why it fails for the mgmt network and to fix the issue instead of making a workarounds?",
      "parentUuid": "25bf3578_ba1f1a41",
      "range": {
        "startLine": 97,
        "startChar": 62,
        "endLine": 97,
        "endChar": 78
      },
      "revId": "b6c1fadb20534a456f57d3ba9a80779ac7094b00",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}